docs/en/sim_sih/index.md

```diff
@@ -113,7 +146,7 @@ In this case you don't need the flight controller hardware.
 To run SIH as SITL:
```

Comment:
Above we say

> The SIH-simulated vehicle can be displayed using [jMAVSim](../sim_jmavsim/index.md) as a visualiser.

jMAVSim is no longer supported. Can we use Gazebo for this?

Reply:
Good question, it is possible, yes but not out of the box. It requires a world with disabled physics were the drone position flows in the reverse direction. Instead simulator -> px4 it needs to be px4 -> simulator.
So Gazebo will need to act as a simple rendering software

Reply:
OK. Issue created - hoping your or Jae can suggest how we might move towards this https://github.com/PX4/PX4-Autopilot/issues/25143